lib/lambda-cfn.js

```diff
@@ -381,6 +381,13 @@ function lambda(options) {
     fn.Properties.MemorySize = 128;
   }
 
+  if (options.runtime) {
```

Comment:
Fixed in https://github.com/mapbox/lambda-cfn/pull/41/commits/495c0d5cf94eed9f9efe4844fe21c5768639e5e3. Previously "runtime" would have been undefined and likely would've errored out when deployed to AWS.

Now the runtime defaults to `nodejs4.3` if not specified. We should keep an eye on if/when AWS Lambda upgrades to a newer version of Node.js and update the default accordingly.